openff/interchange/components/_packmol.py

```diff
@@ -633,15 +667,16 @@ def pack_box(
 
     Notes
     -----
-    Returned topologies may have smaller or larger box vectors than what would be defined by the
-    target density if the box vectors are determined by `target_density`. When calling Packmol, each
-    linear dimension of the box is scaled up by 10%.  However, Packmol by default adds a small
-    buffer (defined by the `tolerance` argument which otherwise defines the minimum distance,
-    default 2 Angstrom) at the end of the packed box, which causes small voids when tiling copies of
-    each periodic image. This void is removed in hopes of faster equilibration times but the box
-    density is slightly increased as a result. These changes may cancel each other out or result in
-    larger or smaller densities than the target density, depending on argument values.
+    Packmol places molecules at random positions in the box, satisfying
+    geometric constraints but no thermodynamic considerations. The resulting
+    configurations need to be subjected to energy minimization and equilibration
+    routines before being used in production simulations.
+
+    Periodic boundary conditions are accounted for during packing if using Packmol version 20.15.0 or newer.
 
+    If Packmol version 20.15.0 or less is used, periodic boundary conditions may not be handled correctly.
+
+    Periodic boundary conditions are accounted for during packing.
```

Comment:
These lines should be covered by the above suggestion but GitHub wouldn't let me 

```suggestion
```